lib/p2p/Pool.ts

```diff
@@ -11,6 +11,7 @@ import { OutgoingOrder, OrderIdentifier, StampedPeerOrder } from '../types/order
 import { Models } from '../db/DB';
 import Logger from '../Logger';
 import { HandshakeState, Address, NodeConnectionInfo } from '../types/p2p';
+import { NodeInstance } from '../types/db';
```

Comment:
I don't think `NodeInstance` is actually needed in this file. I pulled down the branch and removed all references and it compiled fine. Is there another reason for this, or can it just be removed?

Reply:
I used it to make it clear that `connectNodes` can also get a db node entry. I ended up not doing anything with it, but still left this definition. I'll remove it.